### ISSUE TITLE
Add python to the docker container

### ISSUE THREAD

**Issue by Fischy667** [vgist/dockerfiles#32]
It would be nice to have python installed in the container to use the search engines. After every update of the container I have to do it manually.

https://stackoverflow.com/questions/62554991/how-do-i-install-python-on-alpine-linux/62555259#62555259

**Comment by aieu** [comment]
I think you needed python3 for qbittorrent?

**Comment by Fischy667** [comment]
No, qBittorrent is a C++ application. Python is only needed for the search plugins.

**Comment by aieu** [comment]
Ok, I see
